test/ToricVarieties/runtests.jl

```diff
@@ -191,6 +191,13 @@ end
     @test is_projective_space(ntv2) == false
 end
 
+P = polarize(Polyhedron(Polymake.polytope.rand_sphere(5,60)))
```

Comment:
```suggestion
P = polarize(Polyhedron(Polymake.polytope.rand_sphere(5,60; seed=42)))
```
Please add a seed here to make sure the testsuite is not affected by the random number generation.

Reply:
Thank you @benlorenz !